README.md

```diff
@@ -60,6 +60,9 @@ Emoji | Represents | Links to | Comment
 📢 | Talks | the slides/recording/repo/etc.
 🎨 | Design | the logo/iconography/visual design/etc.
 👀 | Reviewed Pull Requests
+💵 | Financial Support | relevant page
+🔍 | Funding Finders
```

Comment:
Hehehe. I feel you. I think I like the first option better. It gives the project maintainer the chance to elaborate further and define what kind of funding opportunities their project pursues the most.